sinch/domains/voice/__init__.py

```diff
@@ -0,0 +1,389 @@
+        enable_dice: bool = None,
+        enable_pie: bool = None
+    ) -> VoiceCalloutResponse:
+        return self._sinch.configuration.transport.request(
+            CalloutEndpoint(
+                callout_method=CalloutMethod.TEXT_TO_SPEECH.value,
+                request_data=TextToSpeechVoiceCalloutRequest(
+                    destination=destination,
+                    cli=cli,
+                    dtmf=dtmf,
+                    domain=domain,
+                    custom=custom,
+                    locale=locale,
+                    text=text,
+                    prompts=prompts,
+                    enableAce=enable_ace,
+                    enableDice=enable_dice,
+                    enablePie=enable_pie
+                )
+            )
+        )
+
+    def conference(
+        self,
+        destination: Destination,
+        conference_id: str,
+        cli: str = None,
+        conference_dtmf_options: ConferenceDTMFOptions = None,
+        dtmf: str = None,
+        conference: str = None,
+        max_duration: int = None,
+        enable_ace: bool = None,
+        enable_dice: bool = None,
+        enable_pie: bool = None,
+        locale: str = None,
+        greeting: str = None,
+        moh_class: str = None,
+        custom: str = None,
+        domain: str = None
+    ) -> VoiceCalloutResponse:
+        return self._sinch.configuration.transport.request(
+            CalloutEndpoint(
+                callout_method=CalloutMethod.CONFERENCE.value,
+                request_data=ConferenceVoiceCalloutRequest(
+                    destination=destination,
+                    conferenceId=conference_id,
+                    cli=cli,
+                    conferenceDtmfOptions=conference_dtmf_options,
+                    dtmf=dtmf,
+                    conference=conference,
+                    maxDuration=max_duration,
+                    enableAce=enable_ace,
+                    enableDice=enable_dice,
+                    enablePie=enable_pie,
+                    locale=locale,
+                    greeting=greeting,
+                    mohClass=moh_class,
+                    custom=custom,
+                    domain=domain
+                )
+            )
+        )
+
+    def custom(
+        self,
+        cli: str = None,
+        destination: Destination = None,
+        dtmf: str = None,
+        custom: str = None,
+        max_duration: int = None,
+        ice: str = None,
+        ace: str = None,
+        pie: str = None
+    ) -> VoiceCalloutResponse:
+        return self._sinch.configuration.transport.request(
+            CalloutEndpoint(
+                callout_method=CalloutMethod.CUSTOM.value,
+                request_data=CustomVoiceCalloutRequest(
+                    cli=cli,
+                    destination=destination,
+                    dtmf=dtmf,
+                    custom=custom,
+                    maxDuration=max_duration,
+                    ice=ice,
```

Comment:
I've simply overlooked it :P it's added now

Reply:
Can you then update the test to reflect it? From what I can see, the property is still defined as a string with no hints to build actions and instructions

Reply:
I think I've forgot to mention ADR-0005-enums.md in this PR review.
It was design decision that was made while I was designing this SDK.

That's why I'm providing Enums, helpers etc. (they are present and explained in documentation and code samples) but I don't enforce them.
Maybe we need to revisit this ADR? Until then I prefer to stick to it.

Reply:
I may have missed it, but since you are mentioning helpers, can you point me where are the SVAML helpers to build actions and instructions for each kind of events? I would like to compare it with what is done in the Node.js SDK

Reply:
My last reply was more abstract (just like mentioned ADR is). It't not about SVAML but the whole SDK design.

Reply:
Thanks for the clarification. Now, back to the original comment regarding SVAML, do you plan to add helpers or will you leave it as is?

Reply:
IMO current solution is fairly simple to use. I don't think there is a need to add more to that.

Reply:
Why a step back about providing SVAML as expected by ticket ?
Seems we need to sinch  about design and SDKs expectations